### DISCUSSION TITLE
"Tile" structure in the analyzed soil moisture field

### DISCUSSION THREAD

**Discussion by tais958** [NASA-LIS/LISF#1307] [Q&A]
Hi, I have done a few experiments with Noah-MP model which assimilates SMAP soil moisture data. I found that there is usually 
"tiles" shown in the 2D map of analyzed soil moisture and suspect it may be due to the coarser grids of SMAP data compared to my model grids. Wondering if anyone has noticed this issue and has came up with a good way to mitigate it? For example, maybe there's a better treatment for soil texture input or option for a soother transition between analyzing grids. Thanks in advance!

**Comment by dmocko** [comment]
HI @tais958 

What version of Noah-MP are you using here?

We have seen this behavior in the regular Noah LSM, where the "slope type" parameter can give a blocky pattern to the soil moisture, particularly for the lower soil moisture levels.  This behavior is from the 1-degree "slope type" source dataset.  To prevent this, we recommend that users set the "fixed slope type:" value in the lis.config for Noah - typically a value of "1" is used.

In Noah-MP, this "slope type" is hard-coded to a value of "1" for all grid points, as written by the Noah-MP LSM developers.

So, that's likely not the source of what you are seeing.  Do you see this more in the upper soil layers, or in lower layers as well?

If you want to use a more modern soil texture input, you can try the ISRIC soil texture data in LDT.

**Reply by tais958** [reply]
Hi @dmocko 

Thank you for sharing your experiences!
I am using Noah-MP version 4.0.1. The blocky patter I saw was over the top layer (0 - 0.1 m). 

It's goo to know that slope type may cause the artificial distribution. However, after looking into this more yesterday, I found that is more likely due to the nature of SMAP data grid resolution (which is 9-km). Since I am running the LSM with 1 km grids, the assimilated SMAP 9-km data gives tile-like increments and the gradients are large enough to appear in the analysis. I also looked at the Open Loop run, and the blocky pattern is absent, which confirms that the increments may be responsible for the artificial gradients. 

Wondering if you have also seen this and anything I could do to mitigate it.

Thank you again!

**Reply by dmocko** [reply]
I have not personally seen this.  @sujayvkumar and/or @LIS-navari - do you have any idea what may be causing this pattern, and how to mitigate it?

@tais958 - can you share a sample image, and also share your SMAP and LSM attribs/pertattribs files?

**Reply by LIS-navari** [reply]
I have not run a 1km test case. 
Could you please share your LIS parameter file and a sample of the tile pattern in the output?

**Reply by tais958** [reply]
Thanks, @dmocko and @LIS-navari ! Please see the example figures and LIS config file as attached. Please let me know if more info is needed for you to diagnose.

![SM_OL_DA_diff](https://user-images.githubusercontent.com/57816398/234377523-7947822d-7eda-4c9c-8abf-84ff74037a3e.png)
[lis.config_noahmp401_smapda_9km.large.txt](https://github.com/NASA-LIS/LISF/files/11325941/lis.config_noahmp401_smapda_9km.large.txt)

**Reply by dmocko** [reply]
Thanks!  Please also share these files:

```
Forcing attributes file:                  ./DA_INPUT/forcing_attribs.txt
Forcing perturbation attributes file:     ./DA_INPUT/forcing_pert_attribs.txt
State attributes file:                    ./DA_INPUT/noahmp_sm_attribs.txt
State perturbation attributes file:       ./DA_INPUT/noahmp_sm_pertattribs.txt
Observation attributes file:              ./DA_INPUT/smap_attribs.txt
Observation perturbation attributes file: ./DA_INPUT/smap_pertattribs.txt
```

**Reply by tais958** [reply]
Thank you. Please find the files as in attached zip file.
[attribs.zip](https://github.com/NASA-LIS/LISF/files/11325991/attribs.zip)

**Comment by LIS-navari** [comment]
Hi @tais958 

I do see a similar pattern in the OL simulation. 
Could you please share your parameter file (the netCDF file from the LDT run)?

Thanks

**Reply by tais958** [reply]
Thanks @LIS-navari! Please check the attached parameter file. 

[ldt.config.noahmp401_params.1km.txt](https://github.com/NASA-LIS/LISF/files/11326385/ldt.config.noahmp401_params.1km.txt)

**Comment by LIS-navari** [comment]
Hi @tais958 

Thanks. I meant the output of LDT after you run this config file. It should be a NetCDF file. 

Thanks

**Reply by tais958** [reply]
Hi @LIS-navari. Sorry for the confusion. I tried to upload but it did not go through. Maybe due to the excessive size. Here's the NetCDF file that was the output from LDT run from my google drive. Please let me know if you can access. Thanks!

Link:
https://drive.google.com/file/d/1Mx_CpoFlIoQABDNDjza2nBBI7pDACkmg/view?usp=sharing

**Comment by LIS-navari** [comment]
Hi @tais958, @dmocko 

The parameter file looks fine. 
How did you downscale the forcing data? 
Maybe this feature is coming from the forcing data.

**Reply by tais958** [reply]
Hi @LIS-navari, below are the relevant options I used when running LIS:

Number of met forcing sources:    2
Blending method for forcings:     "overlay"
Met forcing sources:              "GDAS"  "NLDAS2"
Topographic correction method (met forcing):  "none"  "none"  
Enable spatial downscaling of precipitation:   0 
Spatial interpolation method (met forcing):   "budget-bilinear"  "budget-bilinear"
Spatial upscaling method (met forcing):       "average"  "average"
Temporal interpolation method (met forcing):  "linear"  "linear"

Not sure if this answers your question? Please let me know what I may provide for your assessment. Thanks!

**Reply by tais958** [reply]
Hi @LIS-navari @dmocko. Wondering if you could provide any guidance on this? Thanks for looking into this.
